### ISSUE TITLE
stack level too deep on `bundle exec middleman`

### ISSUE THREAD

**Issue by jenweber** [emberjs/guides#2215]
```
== The Middleman is loading
/usr/local/var/rbenv/versions/2.4.3/lib/ruby/gems/2.4.0/gems/activesupport-4.1.16/lib/active_support/core_ext/numeric/conversions.rb:121: warning: constant ::Fixnum is deprecated
/usr/local/var/rbenv/versions/2.4.3/lib/ruby/gems/2.4.0/gems/activesupport-4.1.16/lib/active_support/core_ext/numeric/conversions.rb:121: warning: constant ::Bignum is deprecated
bundler: failed to load command: middleman (/usr/local/var/rbenv/versions/2.4.3/bin/middleman)
SystemStackError: stack level too deep
```

Replicated locally (mac) and on a cloud9 instance. Using `rbenv`.

**Comment by acorncom** [comment]
@jenweber if you shift to Ruby 2.2 do you still hit this problem? It's working fine on my end when doing so ...

**Comment by jenweber** [comment]
@acorncom yes it seems to work for 2.2.2. Did you have issues with 2.4.3 as well or is it just me? If it worked, can you or @locks do bundle install, try to run it, and post your specs if it works? I'm wondering if you have older versions of dependencies hanging around, old-school-npm-style.

I removed all gems, reinstalled, then tried 2.4.3 and still got the same error.

Specs:
- Mac OS 10
- rbenv 1.1.0
- gem 2.4.5
- ruby 2.2.2 works

**Comment by jenweber** [comment]
Also I don't know why I didn't post this earlier, but I get an infinite loop of this error:

> /usr/local/var/rbenv/versions/2.4.3/lib/ruby/gems/2.4.0/gems/activesupport-4.1.16/lib/active_support/core_ext/numeric/conversions.rb:131:in `block (2 levels) in <class:Numeric>'

**Comment by jenweber** [comment]
I blindly updated a bunch of dependencies and it seems to work now. Will PR. Disclaimer, I have no idea what I'm doing.

**Comment by jenweber** [comment]
closed with #2249
